position/helper.gno

```diff
@@ -28,9 +37,95 @@ func tid(tokenId interface{}) grc721.TokenID {
 	case grc721.TokenID:
 		return tokenId.(grc721.TokenID)
 	default:
-		panic(addDetailToError(
-			errInvalidInput,
-			"helper.gno__tid() || unsupported tokenId type",
-		))
+		panic(newErrorWithDetail(errInvalidInput, "unsupported tokenId type"))
```

Comment:
just curious, can we add what kind of type it had??
something like `unsupported token id type{string/uint64/int/.../}`